manual-deployment.md

```diff
@@ -3,4 +3,167 @@ title: Manual Deployment
+
+### 3. Set up additional nodes
+
+Repeat step 2 for each additional node.
+
+### 4. Use the Built-in SQL Client
+
+Start the built-in SQL client from any machine with the `cockroach` binary. This could be one of the node machines or a different machine. 
+
+~~~ shell
+$ ./cockroach sql --insecure --url=postgresql://root@<node-hostname>:26257?sslmode=disable
+# Welcome to the cockroach SQL interface.
+# All statements must be terminated by a semicolon.
+# To exit: CTRL + D.
+~~~
+
+This command uses the `--url` flag to identify the client user and the hostname and port of the node to connect to. You can also specify these details with distinct flags. For more information, see [Use the Built-In SQL Client](use-the-built-in-sql-client.html).
+
+Once you're connected, run some [SQL statements](learn-cockroachdb-sql.html): 
+
+~~~ shell
+root@:26257> CREATE DATABASE bank;
+CREATE DATABASE
+
+root@:26257> SET DATABASE = bank;
+SET DATABASE
+
+root@:26257> CREATE TABLE accounts (id INT PRIMARY KEY, balance DECIMAL);
+CREATE TABLE
+
+root@26257> INSERT INTO accounts VALUES (1234, DECIMAL '10000');
+INSERT 1
+
+root@26257> SELECT * FROM accounts;
++------+---------+
+|  id  | balance |
++------+---------+
+| 1234 |   10000 |
++------+---------+
+~~~
+
+## Secure Cluster
+
+### 1. Create security certificates
+
+On a machine that won't be part of the cluster, create the CA, node, and client certificates and keys:
+
+~~~ shell
+# Create the CA certificate and key.
+$ ./cockroach cert create-ca --ca-cert=ca.cert --ca-key=ca.key
+   
+# Create the node certificates and keys.
+# For each node, specify any addresses at which the node can be reached.
+$ ./cockroach cert create-node <node1-hostname> <node1-other-hostname> <node1-yet-another-hostname> --ca-cert=ca.cert --cert=node1.cert --key=node1.key
+$ ./cockroach cert create-node <node2-hostname> <node2-other-hostname> <node2-yet-another-hostname> --ca-cert=ca.cert --cert=node2.cert --key=node2.key
+$ ./cockroach cert create-node <node3-hostname> <node3-other-hostname> <node3-yet-another-hostname> --ca-cert=ca.cert --cert=node3.cert --key=node3.key
+
+# Create a certificate and key for each client user, including root. 
+$ ./cockroach cert create-client root --ca-cert=ca.cert --ca-key=ca.key --cert=root.cert --key=root.key
+$ ./cockroach cert create-client <username1> --ca-cert=ca.cert --ca-key=ca.key --cert=username1.cert --key=username1.key
+$ ./cockroach cert create-client <username2> --ca-cert=ca.cert --ca-key=ca.key --cert=username2.cert --key=usename2.key
+~~~
+
+Store the CA key somewhere safe and keep a backup; if you lose it, you will not be able to add new nodes or clients to your cluster.
+
+### 2. Set up the first node
+
+Copy the `cockroach` binary, CA certificate, and node 1 certificate and key to the first machine and then start the node:
+
+~~~ shell
+$ ./cockroach start --ca-cert=certs/ca.cert --cert=certs/node1.cert --key=node1.key --host=<node1-hostname>
```

Comment:
the cert paths are inconsistent here. let's drop `certs/` entirely.
eg:
`$ ./cockroach start --ca-cert=ca.cert --cert=node1.cert --key=node1.key --host=<node1-hostname>`